Repository/AbstractPhpcrRepository.php

```diff
@@ -206,4 +216,12 @@ protected function isGlobbed($string)
      * @param NodeInterface[]
      */
     abstract protected function moveNodes(array $nodes, $query, $targetPath);
+
+    /**
+     * Reorder the node at the given path to be in $position position.
+     *
+     * @param string $sourcePath
+     * @param int $position
```

Comment:
please say here that if position is beyond the end of the list of siblings, the node is to be placed at the end.